### DISCUSSION TITLE
Bun v1.1.31

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#14653] [Announcements]
To install:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.31:
```bash
bun upgrade
```

### Thanks to 19 contributors!

- [@190n](https://github.com/190n)
- [@cirospaciari](https://github.com/cirospaciari)
- [@deiga](https://github.com/deiga)
- [@DonIsaac](https://github.com/DonIsaac)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Electroid](https://github.com/Electroid)
- [@FaSe22](https://github.com/FaSe22)
- [@fel1x-developer](https://github.com/fel1x-developer)
- [@huseeiin](https://github.com/huseeiin)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@nektro](https://github.com/nektro)
- [@paperdave](https://github.com/paperdave)
- [@pfgithub](https://github.com/pfgithub)
- [@refi64](https://github.com/refi64)
- [@RiskyMH](https://github.com/RiskyMH)
- [@Skywalker13](https://github.com/Skywalker13)
- [@snoglobe](https://github.com/snoglobe)
- [@SunsetTechuila](https://github.com/SunsetTechuila)
- [@zackradisic](https://github.com/zackradisic)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.31'>Bun v1.1.31</a>.</em>

**Comment by truongan07** [comment]
I don't know how to implement http2 on native Bun server (Bun.serve) which i am on
